tests/Install-DbaFirstResponderKit.Tests.ps1

```diff
@@ -10,8 +10,7 @@ Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
             $server.Query("CREATE DATABASE $database")
         }
         AfterAll {
-            $server.Query("ALTER DATABASE $database SET OFFLINE WITH ROLLBACK IMMEDIATE")
-            $server.Query("DROP DATABASE IF EXISTS $database")
+            $null = Get-DbaDatabase -SqlInstance $server -Database  $database | Remove-DbaDatabase -Confirm:$false
```

Comment:
This should be reverted back to the original use of `Query()` method because we do not want changes to public functions breaking test on a different command.

Reply:
I would still like to see this reverted back to `Query()` method, regardless of whether the commands will stay around it ensures the test runs at a consistent time.